src/main/java/io/axoniq/axonserver/connector/ReplyChannel.java

```diff
@@ -53,36 +53,45 @@ default void sendLast(T outboundMessage) {
     }
 
     /**
-     * Sends an receipt acknowledgement if one hasn't been sent yet for this instruction. If not explicitly sent, it
-     * will be sent once the {@link #complete()} method is invoked.
-     * <p>
+     * Used to send a positive result, indicating that the incoming message has been handled as expected.
      * If the incoming instruction has no instruction ID, this method does nothing.
+     * This method is deprecated because the {@link #complete()} sends the result.
+     *
+     * @see #complete()
+     * @deprecated
      */
-    void sendAck();
```

Comment:
Sorry, I think I wasn't clear.
This PR removes the `sendAck()` method.

It's *that* method which you cannot simply remove as it's a breaking change.
Hence, I'd expect `sendAck()` to be put back in, but `@deprecated`.

Then, the `sendSuccessResult` would (I assume) default to the `sendAck()` method, as it (AFAIK) replace it.

This line of thinking also applies to the `sendNack` and `sendNack(ErrorMessage)` methods.